addon/components/o-s-s/avatar.hbs

```diff
@@ -0,0 +1,8 @@
+<div class={{this.computedClass}} ...attributes>
+  {{#if this.displayInitial}}
+    <span class="text-style-uppercase">{{@initials}}</span>
+  {{else}}
+    <img src={{this.image}} alt={{t "oss-components.badge.image_alt"}} class={{this.imageClass}}
```

Comment:
you don't need to use a translation for the alt, something like "avatar" should be enough.
Also you shouldn't use the badge.image_alt which has "Badge Icon" as value.